requirements.txt

```diff
@@ -5,6 +5,7 @@ Jinja2==3.0.1
 iso8601==0.1.14
 pytz==2021.1
 aiofiles==0.7.0
+pyproj==2.6.1
```

Comment:
`pyproj` is required here for the tests to pass.
Do `pyproj` or any of the other dependencies here need to be included in `setup.py` for creating a python package in the future?

Reply:
There seems to be one in `feapi/app/`
I think it'll take some experimentation to figure out what the minimum requirements are.